package.json

```diff
@@ -21,7 +21,14 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "npx eslint .",
+    "lintf": "npx eslint --fix .",
+    "style": "npx stylelint \"**/*.{css,scss}\"",
+    "stylef": "npx stylelint --fix \"**/*.{css,scss}\"",
+    "format": "prettier --write \"src/**/*.{js, jsx}\"",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```

Comment:
very clever!
It's the first time I see this, I'll benchmark it for future use.
With your permission.

Reply:
sure you can use it, it is ver useful @eduardosancho